authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/RestSourceUserResource.kt

```diff
@@ -64,7 +64,7 @@ class RestSourceUserResource(
         @QueryParam("source-type") sourceType: String?,
         @QueryParam("size") pageSize: Int?,
         @DefaultValue("1") @QueryParam("page") pageNumber: Int,
-    ): RestSourceUsers {
```

Comment:
Why not just add a `metadata` element to restsourceusers, including page, page size and total items? If we need metadata in another element, we can reuse the metadata type.